contrib/demo/runDemoScripts.sh

```diff
@@ -15,8 +15,8 @@
 # limitations under the License.
 
 CURRENT_DIR="$(pwd)"
-DEMO_ROOT="$(cd $(dirname "${BASH_SOURCE}") && pwd)"
-KCP_ROOT="$(cd ${DEMO_ROOT}/../.. && pwd)"
+DEMOS_ROOT="$(cd $(dirname "${BASH_SOURCE}") && pwd)"
+KCP_ROOT="$(cd ${DEMOS_ROOT}/../.. && pwd)"
```

Comment:
Can you break this into a separate PR? Seems unrelated to the rest of the content in here

Reply:
Done in PR #381